WordPress/Classes/ViewRelated/NUX/SignupEpilogueTableViewController.swift

```diff
@@ -21,8 +21,8 @@ class SignupEpilogueTableViewController: UITableViewController, EpilogueUserInfo
 
     // MARK: - Properties
 
-    open var dataSource: SignupEpilogueTableViewControllerDataSource?
-    open var delegate: SignupEpilogueTableViewControllerDelegate?
+    open weak var dataSource: SignupEpilogueTableViewControllerDataSource?
+    open weak var delegate: SignupEpilogueTableViewControllerDelegate?
```

Comment:
https://github.com/wordpress-mobile/WordPress-iOS/blob/48b8a391394f2b4aee9b886abe21f62a521c0a81/WordPress/Classes/ViewRelated/NUX/SignupEpilogueViewController.swift#L71-L79

I'm not sure if a view controller would hold strong references to segue instances, which would cause a retain cycle in our case here. But considering how this `delegate` and `dataSource` are used, they should be weak references anyways.

Reply:
Yes, I jumped on this memory leak in other investigations.